.github/labeler.yml

```diff
@@ -119,6 +119,9 @@ area/storage:
       - any-glob-to-any-file:
           - content/storage/**
 
+area/accounts:
+  - content/accounts/**
+
```

Comment:
Looks like this uses the old configuration format for the labeler action. It should take the same form as the others.